src/os_view.eliom

```diff
@@ -3,10 +3,25 @@
    and integrate it in your app to customize the behaviour according to
    your needs. *)
 
+(** This module defines functions to create password forms, connection forms,
+ * settings buttons and other stuff common contents to easily create content
+ * and start your application. Feel free to add other contents you need.
```

Comment:
Maybe something like "... settings buttons and other kinds of content commonly arising in applications"?